ejb/src/test/java/org/wildfly/httpclient/ejb/AsyncInvocationTestCase.java

```diff
@@ -86,7 +87,11 @@ public void testSimpleAsyncException() throws Exception {
         }
     }
 
+    /*
+     * TODO: undo ignore when we find out how to manage the now missing JSESSIONID with cancellation
+     */
     @Test
+    @Ignore
```

Comment:
Oh, I see now. It would be good to have a follow up Jira, unless you plan to tackle this at this Jira only, after this PR is merged.